### ISSUE TITLE
Accuracy is decresing after every epoch

### ISSUE THREAD

**Issue by AmmarahFarooqnust** [facebook/fb.resnet.torch#112]
I am using the code for a different dataset... my accuracy starts decreasing after 3,4 epochs. I found same result from both using pretrained model and training new one. any idea what could be the issue? thanx in advance
